index.js

```diff
@@ -0,0 +1,75 @@
+import Book from './modules/books.js';
+import Storage from './modules/storage.js';
+import displayDynamic from './modules/displaydynamic.js';
+import DateTime from './modules/luxon.js';
+
+document.addEventListener('DOMContentLoaded', displayDynamic.displayBooks);
+document.querySelector('#form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const inputTitle = document.querySelector('.title').value;
+  const inputAuthor = document.querySelector('.author').value;
+  const book = new Book(inputTitle, inputAuthor);
+  displayDynamic.addBooksCollection(book);
+  Storage.addBook(book);
+  displayDynamic.clearFields();
+});
+
+document.querySelector('#list').addEventListener('click', (e) => {
+  if (e.target.className === 'remove') {
+    const book = e.target.parentElement;
+    Storage.removeBook(book);
+    displayDynamic.deleteBook(e.target);
+  }
+});
+
+const timeAndDate = document.querySelector('.time');
```

Comment:
- Good job here, but unfortunately your date is not showing up. Kindly use `#time` instead of `.time` in your `querySelector()` method; it should work as expected. :+1: